src/components/Dialogs/PostDetail.js

```diff
@@ -1,78 +1,70 @@
-import _ from 'lodash';
-import React, { useState } from "react";
-import { Link } from "react-router-dom";
-import {
-  Paper,
-  Container,
-  Grid,
-  Typography,
-} from "@material-ui/core";
+import _ from "lodash"
+import React, { useState } from "react"
+import { Link } from "react-router-dom"
+import { Paper, Container, Grid, Typography } from "@material-ui/core"
 
-import PostReply from "./PostReply.js";
-import UpvoteDownVote from "./UpvoteDownvote.js";
-import ReplyForm from "./ReplyForm.js";
-import { Replay } from '@material-ui/icons';
+import PostReply from "./PostReply.js"
+import UpvoteDownVote from "./UpvoteDownvote.js"
+import ReplyForm from "./ReplyForm.js"
 
 export default function PostDetail(props) {
-
   const currentUser = { username: "CurrentUser" }
 
   const post = {
     id: 2,
-    title: 'These are great resources for learning Vue.js',
-    name: 'Scottie Wall',
+    title: "These are great resources for learning Vue.js",
+    name: "Scottie Wall",
     postText:
-      'ya ya standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+      "ya ya standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
     replies: [{ id: 3 }],
     flag: 0,
-    date: 'Nov 16, 2019',
-    time: '10:05pm',
-    votes: 3
+    date: "Nov 16, 2019",
+    time: "10:05pm",
+    votes: 3,
   }
 
   const replies = [
     {
       id: 1,
       postId: 1,
-      username: 'user1',
-      text: 'this is a comment to the post',
-      notification: ''
+      username: "user1",
+      text: "this is a comment to the post",
+      notification: "",
     },
     {
       id: 2,
       postId: 1,
-      username: 'user2',
-      text: 'another comment to the post',
-      notification: ''
+      username: "user2",
+      text: "another comment to the post",
+      notification: "",
     },
     {
       id: 3,
       postId: 2,
-      username: 'user1',
-      text: 'yet another comment to the post',
-      notification: ''
-    }
+      username: "user1",
+      text: "yet another comment to the post",
+      notification: "",
+    },
   ]
 
   const [postState, setPostState] = useState(post)

```

Comment:
Since we're not using `setPostState` you can remove this line completely and replace all instances of `postState` with `post`